### PR TITLE
Add package name to log

### PR DESCRIPTION
**Pull request by dd-jy**
## Description

Add package name to log

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_util/set_log.py [hunk 1]
```diff
@@ -55,6 +64,7 @@ def init_log(log_file, create_file=True, stream_log_level=logging.INFO,
             logger.addHandler(file_handlder)
 
         logger.propagate = False
+    logger = CustomAdapter(logger, main_package_name.upper())
```
Comment (soimkim) [inline src/fosslight_util/set_log.py:67]: @dd-jy , 여기에서 main_package_name.upper().replace ("FOSSLIGHT_","") 은 어때요?
조금이라도 LOG 길이를 줄여보고 싶어서요
`[FOSSLIGHT_SCANNER] -> [SCANNER]`

Comment (dd-jy) [reply src/fosslight_util/set_log.py:67]: @soimkim , fosslight 제거할까 하다가 fosslight_scanner가 scanner로 되는게 알아보기 힘들거 같아서 추가하지 않았는데요. 괜찮을까요?

Comment (soimkim) [reply src/fosslight_util/set_log.py:67]: @dd-jy 님의 의견을 따르겠습니다! 골라주십셔~ 😉

Comment (dd-jy) [reply src/fosslight_util/set_log.py:67]: @soimkim 직관적으로 FOSSLight 붙이는걸로 가시죠~

**Pull request merged**